### DISCUSSION TITLE
Does the DockerHub changes affect asreview? If so, how to handle them?

### DISCUSSION THREAD

**Discussion by abelsiqueira** [asreview/asreview#1353] [Ideas]
Docker announced last week that [it is deleting Open Source organisations](https://blog.alexellis.io/docker-is-deleting-open-source-images/). Is asreview affected by it?
They have better explained the changes in the post ["We apologize. We did a terrible job announcing the end of Docker Free Teams"](https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/), and their suggested alternative (aside from paying) is their own [Open Source Program](https://www.docker.com/community/open-source/application/).

**Comment by J535D165** [comment]
I think we are affected according to the post. 

<img width="1264" alt="image" src="https://user-images.githubusercontent.com/12981139/226319129-35e5b1fa-4a79-48f6-9081-11bc052c6dd6.png">

Maybe we should sign up for the Open Source Program.

**Reply by abelsiqueira** [reply]
This seems like the simplest approach. One alternative being discussed (to prevent DockerHub dependency) is to store the image somewhere else, like the [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) or even Zenodo (like [SPEX](https://spex-xray.github.io/spex-help/getstarted/docker.html#step-2-download-the-spex-docker-image-from-zenodo)), but these approaches change the user experience from the default Docker experience.

**Reply by jteijema** [reply]
I do like the alternatives offered in the article. Personally I'd be in favour of migrating to _Zenodo_, over the _Open Source Program_ after the way they handled this. Assuming migration won't be too disruptive.

**Reply by J535D165** [reply]
Let's go for GitHub. GitHub is officially supported by the UU. (DockerHub and Zenodo are not). 

Let me know what you need from me @abelsiqueira.

**Reply by abelsiqueira** [reply]
The latest update on pull request #1341 is using the GitHub registry.
I think it should be good to manually re-push the older image to the DockerHub, so it is back to a working state. I think that installing `buildx` should enable you to build it cross-platform.

**Reply by abelsiqueira** [reply]
@J535D165, I found one thing that still needs to be done: Mark the packages as public.
I think it is in this link: 

https://github.com/organizations/asreview/settings/packages

However, I am not sure because I did not have to do it for my repo. I think that since asreview is an organization, it needs this.

**Reply by J535D165** [reply]
Thanks, checkes the public box!

**Reply by J535D165** [reply]
Apparently, there was also another checkbox to check. Now it is public: https://github.com/asreview/asreview/pkgs/container/asreview

**Reply by abelsiqueira** [reply]
Tested and it works.
